module-platform/linux/tests/gendisktestimg.sh

```diff
@@ -1,25 +1,34 @@
 #!/bin/bash -e
-# Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
 # For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
-#Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+#Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
 #For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 if [ $# -ne 1 ]; then
 	echo "Error! Invalid argument count"
 	exit -1
 fi
+
+if [ -z ${L_GIT_DIR+unbound} ] && [ -z $(export L_GIT_DIR=$(git rev-parse --show-toplevel 2>/dev/null)) ]; then
+  export L_GIT_DIR=$(readlink -f "$(dirname "$0")/../../..");
+fi
```

Comment:
This line of code translates to this:
> **IF** `L_GIT_DIR` is unbound **AND** export results in a null string **THEN** export a relative location

Reply:
Without this, when you run a script using `sudo` it will spit out
```
fatal: unsafe repository ('path2repo/MuditaOS' is owned by someone else)
To add an exception for this directory, call:

        git config --global --add safe.directory path2repo/MuditaOS
```